backend/modules/user/controllers/auth/authenticateUser.ts

```diff
@@ -1,5 +1,6 @@
 import { Response, Request } from "express";
-import { loginUser } from "../../services/userAuthentication.ts";
+import userAuthentication from "../../services/userAuthentication.ts";
```

Comment:
This is the point about the imports in general. When I read through the functions I need to check import to see where the function calls are coming from.
If you use default imports you can name the import something like `userAuthService` instead and then in code when you read through the controller function it's clear that this function is a service function:

```
userAuthService.loginUser
```